JS/Lesson19/onder/exercises.js

```diff
@@ -1,8 +1,163 @@
+console.log('-------------Exercise 1------------------');
```

Comment:
Hi Önder, you'll need to remove this change to lesson 19 file from this PR, please how we did similar thing together a few weeks ago :)